apache-spark-master/build/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ubuntu
+
+MAINTAINER Frank Austin Nothaft, fnothaft@berkeley.edu
+
+RUN apt-get update && \
+  apt-get install -y \
+    python \
+    libnss3 \
+    openjdk-8-jdk \
+    curl
+
+COPY download.sh /home/
+RUN sh /home/download.sh
```

Comment:
It takes ~3 seconds to do the download and ~55 seconds to do the git clone. Given that the container takes 15 minutes to build I'm inclined to go for the clone since it simplifies the code. I don't have a strong preference though.